### ISSUE TITLE
NTR: Thin basement membranes (level 3)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6290]
Thin basement membranes (level 3)
parent:Abnormalities of the glomerular basement membrane (NEW TERM)
def:Decreased thickness of the GBM, measured from endothelial to visceral epithelial plasma membrane and mainly attributable to a decrease in thickness of the lamina densa, generally to an overall thickness >2 SD less than that of the normal mean GBM thickness for health age- and sex matched individuals.  May be focal or diffuse, although the term “thin GBMs” generally implies thinning of >50% of GBMs @avizrosenberg @yongqunh

**Comment by pnrobinson** [comment]
This term existed, https://hpo.jax.org/app/browse/term/HP:0012577, I have updated the definition and label and hierarchy.
